core/pom.xml

```diff
@@ -897,7 +896,7 @@
 						<phase>process-classes</phase>
 						<configuration>
 							<proc>only</proc>
-							<compileSourceRoots>${project.build.directory}/generated-sources/delombok</compileSourceRoots>
+							<compileSourceRoots>${project.build.outputDirectory}</compileSourceRoots>
```

Comment:
Ah, did the output of this plugin get overwritten by the delombok step afterwards?

Reply:
No, this directory only exists when running maven with the delombok step. Normal class info can be used here, but both paths could be added if needed.